### DISCUSSION TITLE
How to call `custom.js` functions from a template?

### DISCUSSION THREAD

**Discussion by Sininho** [stdword/logseq13-full-house-plugin#41] [Q&A] [answered]
Hello. 
I have global functions defined in  logseq's custom.js file, and I want to use them in the javascript environment of a full house template. Is it possible?

As a quick example, I have in the the custom.js: 

```
window.globalFunction = function() {
  alert('This is a global function');
}
```

I can call it from the console, but if I try to to it in a template I get: 

```
line 1
 >> 1| ``{window.globalFunction()}``

window.globalFunction is not a function
```

Is there a way to do it, or isn't it supported? 

Thank you again for the fantastic you have done with the fullhouse plugin, it is amazing! :blush:

**Comment by stdword** [comment, marked as answer]
Sure, it works. Just use `top` instead of `window`:
```javascript
top.globalFunction()
```

**Reply by Sininho** [reply]
_That_ easy!? I should have asked earlier instead of spending so much time trying to figure it out myself :sweat_smile: 
Thanks so much, you are the best :+1:
